pika/adapters/select_connection.py

```diff
@@ -367,11 +373,11 @@ def _process_fd_events(self, fd_event_map, write_only):
                 handler = self._fd_handlers[fileno]
                 handler(fileno, events, write_only=write_only)
 
-    def flush_outbound(self):
+    def _flush_outbound(self):
         """Call the state manager who will figure out that we need to write
             then call the poller's poll function to force it to process events.
         """
-        self._()
+        self._manage_event_state()
```

Comment:
Ah right, apologies, yes that is garbage. Never gets called and should just
be deleted.

On Thu, 7 May 2015 at 10:31 vitaly-krugl <notifications@github.com> wrote:

> In pika/adapters/select_connection.py
> <https://github.com/pika/pika/pull/559#discussion_r29836935>:
>
> >          """Call the state manager who will figure out that we need to write
> >              then call the poller's poll function to force it to process events.
> >          """
> > -        self._()
> > +        self._manage_event_state()
>
> @wjps <https://github.com/wjps>, but this _flush_outbound is inside
> SelectPoller, which is not subclassed from BaseConnection. I think that
> you may have accidentally pasted _flush_outbound() into SelectPoller
> class. The legitimate _flush_outbound is in SelectConnection.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pika/pika/pull/559/files#r29836935>.
>